tests/functional/test_install.py

```diff
@@ -1850,7 +1850,7 @@ def test_install_sends_client_cert(install_args, script, cert_factory, data):
 
     assert server.mock.call_count == 2
     for call_args in server.mock.call_args_list:
-        environ, _ = call_args.args
+        environ, _ = call_args[0]
```

Comment:
Maybe add a note to change it back to `call_args.args` once we drop everything <3.8? It’d also serve as a readability cue for future maintainers.